bedrock/security/templates/security/bug-bounty/faq.html

```diff
@@ -190,10 +190,7 @@ <h1 itemprop="name" class="title-shadow-box">Bug Bounty Program FAQ</h1>
             the system requirements for the
```

Comment:
The "the" in this sentence is incorrect but, it is already out there so, not sure if we can remove it at this point.